packages/graphback-cli/src/cliName.ts

```diff
@@ -0,0 +1 @@
+export const CLI_NAME = "graphback "
```

Comment:
WDYT? `commandRoot`

Reply:
I would avoid space here and use it in actual templates

Reply:
@wtrocki there's an issue with space as if I delete it and add space to print out in [init.ts]( 
https://github.com/aerogear/graphback/pull/313/files#diff-7b17caceb59bc1f2a0a7bbf6a6a954afR96) graphback print outs will look fine but graphql generate command will look like this:
```
graphql backend- generate
```
while it should be :
```
graphql backend-generate
```
Not sure if there's any workaround that if this is the approach I took.

Reply:
Yes. I think we might need to do some backend subcomand at some poit. Merge it as it is